code/game/objects/_items.dm

```diff
@@ -95,7 +95,7 @@
 	// Works similarly to worn sprite_sheets, except the alternate sprites are used when the clothing/refit_for_species() proc is called.
 	var/list/sprite_sheets_obj = list()
 
-	//Items are not dense typically
+	// Items are not dense typically
```

Comment:
is there some problem with not having a space here?

Reply:
Every comment in that file has space after //